### ISSUE TITLE
[Improvement] 24h time support

### ISSUE THREAD

**Issue by dosch** [Swing-team/pendulums-web-client#5]
heya,

24h time support would be really nice.
Thnx and keep up the good work!

**Comment by dosch** [comment]
Hey, sure!
When adding an event, I can only choose AM/PM in the time selecter. An 24h option would be a nice to have here.

**Comment by mohammadrafigh** [comment]
sure we will consider it.

**Comment by moqmar** [comment]
It seems like this is a browser configuration issue - I can correctly enter 24h time, or have you fixed this in the meantime?

**Comment by mohammadrafigh** [comment]
Good point @moqmar. No we didn't fix it but we will consider this in further releases. We are currently working on Pendulums 2.0 which will bring a lot of new features and better UX/UI, so our task stack is full for now. ;)
